clang/include/clang/Driver/Options.td

```diff
@@ -4706,6 +4706,10 @@ def image__base : Separate<["-"], "image_base">;
 def include_ : JoinedOrSeparate<["-", "--"], "include">, Group<clang_i_Group>, EnumName<"include">,
     MetaVarName<"<file>">, HelpText<"Include file before parsing">,
     Visibility<[ClangOption, CC1Option]>;
+def include_footer : Separate<["-"], "include-footer">, Group<clang_i_Group>,
+    Visibility<[ClangOption, CC1Option]>,
```

Comment:
There is no intention of having this option be visible to the end user due to the limited usage model, I think `CC1Option` only should be OK.